docker/run.sh

```diff
@@ -4,10 +4,14 @@
 CACHE_SIZE="${TAKAHE_NGINX_CACHE_SIZE:-1g}"
 sed -i s/__CACHESIZE__/${CACHE_SIZE}/g /takahe/docker/nginx.conf
 
+# Enable auto-reload when in DEBUG mode
+[ "${TAKAHE_DEBUG}" = "true" ] && GUNICORN_RELOAD="--reload"
+
 # Run nginx and gunicorn
 nginx -c "/takahe/docker/nginx.conf" &
 
-gunicorn takahe.wsgi:application -b 0.0.0.0:8001 &
+echo $GUNICORN_RELOAD 4
+gunicorn takahe.wsgi:application -b 0.0.0.0:8001 $GUNICORN_RELOAD &
```

Comment:
`GUNICORN_CMD_ARGS` exists to override the options

Reply:
We should probably make it the default in the docker-compose for dev

Reply:
I am hesitant to use `GUNICORN_CMD_ARGS` since it is already in use by the Dockerfile and likely to be the first thing people reach for (I use it for configuring my deployment environment on fly.io as well).

So I will set a new env var in docker compose for this specific purpose. Commit incoming.